src/test/java/com/diffblue/deeptestutils/regression/ClassWithConstructor.java

```diff
@@ -0,0 +1,12 @@
+package com.diffblue.deeptestutils.regression;
```

Comment:
:bulb: It would be super awesome if you were you to add tests for what happens when creating abstract classes and interfaces  since that code is a complicated under tested mess

Reply:
I will do this in the follow-up PR for https://diffblue.atlassian.net/browse/TG-5895, as that will fix the exception wrapping for interfaces and abstract classes.

Reply:
Tests added in https://github.com/diffblue/deeptest-utils/pull/61, with an additional test enabled in https://github.com/diffblue/deeptest-utils/pull/62.